### DISCUSSION TITLE
Why not uncompressed audio quality

### DISCUSSION THREAD

**Discussion by openaspace** [jamulussoftware/jamulus#3238] [Ideas]
Hello, why not allow uncompressed audio as an option in the server?

**Comment by dcorson-ticino-com** [comment]
No one has anything against uncompressed audio.
The problem is just to find someone who wants to do the work in a very complicated portion of the program.
Would you like to do it ?

**Reply by openaspace** [reply]
an application with this low latency and power cannot fully deliver uncompressed audio quality.. Especially in the music field, not being able to enjoy the superior quality of current high-end audio systems. Now, with fiber optic connections widely available and upload speeds of at least 250mbps....

**Reply by jujudusud** [reply]
Shame?
Jamulus was developed by a person at a time when fibre was not available to the general public. And it's not necessarily available everywhere even today...
Secondly, your tone is inappropriate and I don't think that's a good way of getting a team together to develop new functions for Jamulus.
Finally, you don't answer the question put to you.
Would you like to add uncompressed audio quality to Jamulus?

**Reply by openaspace** [reply]
Really sorry, translation error..!
I was mean "It's a pity".. for the future potential

**Reply by openaspace** [reply]
> Jamulus was developed by a person at a time when fibre was not available to the general public. And it's not necessarily available everywhere even today...
> Secondly, your tone is inappropriate and I don't think that's a good way of getting a team together to develop new functions for Jamulus.
> Finally, you don't answer the question put to you.
> Would you like to add uncompressed audio quality to Jamulus?

I could never do it.. I speak as a user who uses it to play every day..

Sorry again for the translation error...

**Reply by jujudusud** [reply]
No problem, I'm french and sometime I use a word at a wrong place. ;-)

**Comment by Andy081549** [comment]
I just wanted to ask the same without having an idea how complicated this would be. Bandwith usage would rise, ok, but that should not be a problem (48k, Stereo: 1,54 MBit/s) - I can feed 10 clients simultineously in stereo through a mid range DSL line (10 MBit).

Another benefit beneath audio qualitiy could be reduced processing time due to the lack of compression (?).

Maybe there is a fork of Jamulus with that option?

**Reply by dcorson-ticino-com** [reply]
My comment from above remains valid:
"The problem is just to find someone who wants to do the work in a very complicated portion of the program.
Would you like to do it ?"
